scripts/install.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+RELEASE_VERSION=${1-}
+
+if [[ -z "${RELEASE_VERSION}" ]]; then
+    echo -e "ERROR: No release version given\n"
+    echo -e "Usage:\n"
+    echo -e "  $0 release_version\n"
+    echo "Release version format is X.Y.Z where X, Y, and Z are non-negative integers"
+    echo "You can find a list of epoch releases at https://github.com/aeternity/epoch/releases"
+    exit 1
+fi
+
+TEMP_RELEASE_FILE=${TEMP_RELEASE_FILE:=/tmp/epoch.tgz}
+TARGET_DIR=${TARGET_DIR:=$HOME/aeternity/node}
+
+install_prompt () {
+    echo -e "\nATTENTION: This script will delete the directory ${TARGET_DIR} if it exists. You should back up any contents before continuing.\n"
+    read -p "Continue (y/n)?" inputprerunchoice
+    case "$inputprerunchoice" in
+        y|Y )
+            echo "Continuing..."
+            ;;
+        n|N )
+            echo "Exiting..."
+            exit 0
+            ;;
+        * )
+            echo "Invalid input..."
+            install_prompt
+            ;;
+    esac
+}
+
+install_deps_ubuntu() {
+    OS_RELEASE=$(lsb_release -r -s)
+    echo -e "\nPrepare host system and install dependencies ...\n"
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    sudo apt-get install -y curl libssl1.0.0
+
+    if [[ "$OS_RELEASE" = "16.04" ]]; then
+        sudo apt-get install -y build-essential
+        wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz
+        tar -xf libsodium-1.0.16.tar.gz && cd libsodium-1.0.16 &&
+        ./configure && make && sudo make install && sudo ldconfig
+        cd .. && rm -rf libsodium-1.0.16 && rm libsodium-1.0.16.tar.gz
```

Comment:
Lots of hardcoded `1.0.16` - would be nice to group them in a variable though may be debatable so ok.

Reply:
it was copy-pasta :) see 99b5a776